Sources/ProjectDescription/Dependencies/SwiftPackageManagerDependencies.swift

```diff
@@ -14,21 +14,28 @@ public struct SwiftPackageManagerDependencies: Codable, Equatable {
     // Additional settings to be added to targets generated from SwiftPackageManager.
     public let targetSettings: [String: SettingsDictionary]
 
+    /// Custom project configurations to be used for projects generated from SwiftPackageManager.
```

Comment:
This is the only documentation users will have for this property. Let's add an example use-case here. It's ok for the documentation comments to be long 🙂